cli/flox-rust-sdk/proptest-regressions/models/manifest/typed.txt

```diff
@@ -0,0 +1,8 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc 764a939774de015858b38c696bcccb64476e02d42c2aa408e642cd299ea1c09c # shrinks to manifest = Manifest { version: Version { value: 1 }, install: ManifestInstall({}), vars: ManifestVariables({}), hook: ManifestHook { on_activate: None }, profile: ManifestProfile { common: None, bash: None, zsh: None, fish: None, tcsh: None }, options: ManifestOptions { systems: None, allow: Allows { unfree: None, broken: None, licenses: [] }, semver: SemverOptions { allow_pre_releases: None }, cuda_detection: None }, services: ManifestServices({}), build: ManifestBuild({}), containerize: None, include: Include { environments: [Remote] } }
+cc f0f39a3f29504c43451dc7b415a5464e978806fec04cd9ab4ad4d64215a14aa6 # shrinks to manifest = Manifest { version: Version { value: 1 }, install: ManifestInstall({}), vars: ManifestVariables({}), hook: ManifestHook { on_activate: None }, profile: ManifestProfile { common: None, bash: None, zsh: None, fish: None, tcsh: None }, options: ManifestOptions { systems: None, allow: Allows { unfree: None, broken: None, licenses: [] }, semver: SemverOptions { allow_pre_releases: None }, cuda_detection: None }, services: ManifestServices({}), build: ManifestBuild({}), containerize: None, include: Include { environments: [Remote] } }
```

Comment:
(non-blocking, curious) These shrunk cases are the same. Does that mean that the tests failed at some point while you were working on this but it's probably not related to these cases and that they aren't terribly useful to keep?

Reply:
If you have eagle eyes you’ll notice that we don’t have a `Remote` variant, so this was from a WIP step and can be removed.

Reply:
Fixed